packages/scanner/src/cli/scan/singleScan.ts

```diff
@@ -70,4 +71,19 @@ export default async function singleScan(options: SingleScanOptions): Promise<vo
       numChecks / (elapsed / 1000.0)
     )} checks/sec)`
   );
+
+  const { checks } = scanResults;
+
+  Telemetry.sendEvent({
+    name: 'scan:completed',
+    properties: {
+      checks: checks.join(', '),
```

Comment:
I think we may just want unique rule names here. Checks can include properties, which when represented as JSON and then joined into a comma-delimited string is going to be pretty nasty. Really we just want to know which rules the user has enabled, anyway.

Reply:
`Check.toString()` doesn't represent it as JSON (see example above), but ok.